### DISCUSSION TITLE
Enable schedule in dashboard

### DISCUSSION THREAD

**Discussion by ivorynoise** [iSoron/uhabits#1354] [Feature Requests]
Ability to see time/schedule can help to track, sort and execute habits efficiently. Psychologically, if a habit is seen daily with schedule, then it is more probable that it will be executed more frequently. Also, it helps to schedule the day. 

<img src="https://user-images.githubusercontent.com/11216794/163919656-130f12d9-26ee-43b8-8633-eb9e457a07b0.png" width="200" height="400">

In the above image, the idea is to show the schedule below the habit name.

**Comment by hiqua** [comment]
Are you talking about https://github.com/iSoron/uhabits/discussions/372 or is this something else? What's the missing text you'd like to see, with an example?
